assets/pages/new-contributions-landing/components/ContributionFormContainer.jsx

```diff
@@ -98,6 +99,7 @@ function ContributionFormContainer(props: PropTypes) {
     : (
       <div className="gu-content__content">
         <h1 className={headerClasses}>{countryGroupDetails.headerCopy}</h1>
+        { props.countryGroupId === 'UnitedStates' && <ContributionUsTicker /> }
```

Comment:
this is quite a nice way of conditionally rendering, but having it right next to a different way of achieving the same thing feels a bit jarring. maybe we should stick to one?

Reply:
updated.